01.data_download.py

```
@@ -0,0 +1,141 @@
+
+
+#%% download and unzip original data
+basefile_url = 'https://aistages-api-public-prod.s3.amazonaws.com/app/Competitions/000363/data/data.tar'
+basefile_filename = 'basefile.tar'
+basefile_path = os.path.join(rawdata_dir, basefile_filename)
+
+if not os.path.exists(basefile_path):
+    wget.download(basefile_url, basefile_path)
+    with tarfile.open(basefile_path, 'r') as tar:
+        tar.extractall(path=rawdata_dir)
+else: pass
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)